scripts/migration/migrate_retraction_dates.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Script to migrate retracted registrations so that their date modified is date of retraction."""
+
+import sys
+import logging
+
+from modularodm import Q
+from website.models import Node, NodeLog
+
+from website.app import init_app
+from scripts import utils as script_utils
+
+logger = logging.getLogger(__name__)
+
+
+def do_migration(logs):
+
+    # ... perform the migration using a list of logs ...
+    for log in logs:
+        when = log.date
+        registration = Node.load(log.params['registration'])
+
+        registration.update_fields(date_modified=when)
+        registration.save()
+
+        logger.warning("{} date updated to {}".format(registration, when))
+
+
+def get_targets():
+    # ... return the list of logs whose registrations we want to migrate ...
+    query = Q('action', 'eq', "retraction_approved")
```

Comment:
Consider serializing the set of targets into a list of `{<_id>: <correct_date>}` mappings.